aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -535,13 +535,10 @@ void TensorIterator::select_all_keeping_dim(int start_dim, IntArrayRef indices)
 }
 
 TensorIterator TensorIterator::binary_op(Tensor& out, const Tensor& a,
-    const Tensor& b, bool check_internal_overlap) {
+    const Tensor& b, bool check_overlap) {
```

Comment:
nit: `check_mem_overlap` is a little more descriptive

Reply:
addressed